### DISCUSSION TITLE
Metric system as default game option

### DISCUSSION THREAD

**Discussion by scarf005** [cataclysmbn/Cataclysm-BN#2517] [Polls]
# There's no changes for users already playing BN

it's only default value changes.

## For users who newly installed BN

- default interface option will be changed to metric system (km/h, celsius, 24h). 
- they can be reverted to US customary unit (fahrenheit, lbs, etc) anytime on the `interface` tab of game settings.
- game's default volume unit already is liters, so it would stay regardless of poll result.

see #2510 for details.
poll will be open from `2023-03-31 23:11 UTC+09:00` to `2023-04-08 12:00 UTC+09:00`.

**Comment by Coolthulhu** [comment]
Alternative: try to get it from current system locale if it isn't set to anything.

**Reply by scarf005** [reply]
i'm not against that, but we could do #2510 first then merge DDA changes later.
